src/devtools/client/debugger/src/actions/ui.js

```diff
@@ -219,3 +219,14 @@ export function refreshCodeMirror() {
     codeMirror.on("refresh", handler);
   };
 }
+
+export function setFullTextQuery(query) {
+  return { type: "SET_FULLTEXT_SEARCH", query };
+}
+
+export function focusFullTextInput() {
+  return ({ dispatch }) => {
+    dispatch({ type: "FOCUS_FULLTEXT_SEARCH", focus: true });
+    setTimeout(() => dispatch({ type: "FOCUS_FULLTEXT_SEARCH", focus: false }), 100);
```

Comment:
What's this doing??

Reply:
Seems like maybe this would be better in the `onFocus` of the text input? If what you're trying to do is reset it?

Reply:
clever.